003_app_L24.py

```
@@ -0,0 +1,164 @@
+                print('('+str(counter)+') Sending SMS to ' + number)
+                a = cmd_res('adb shell service call isms 7 i32 1 s16 \"com.android.mms\" s16 \"'+number+'\" s16 \"null\" s16 \"'+SMS+'\" s16 \"null\" s16 \"null\"')
+                time.sleep(2)
+            elif android_version == 'Android Marshmallow':
+                counter += 1
+                print('('+str(counter)+') Sending SMS to ' + number)
+                a = cmd_res('adb shell service call isms 7 i32 1 s16 \"com.android.mms\" s16 \"'+number+'\" s16 \"null\" s16 \"'+SMS+'\" s16 \"null\" s16 \"null\"')
+                time.sleep(2)
+            elif android_version == 'Android Lollipop':
+                counter += 1
+                print('('+str(counter)+') Sending SMS to ' + number)
+                a = cmd_res('adb shell service call isms 9 s16 \"com.android.mms\" s16 \"'+number+'\" s16 \"null\" s16 \"'+SMS+'\" s16 \"null\" s16 \"null\"')
```

Ruff (0.14.5)

Local variable `a` is assigned to but never used

Remove assignment to unused variable `a`

(F841)